samples/dci-cargo/dcisample_a/src/main/java/org/qi4j/sample/dcicargo/sample_a/bootstrap/sampledata/BaseDataService.java

```
@@ -146,6 +147,12 @@ public void createBaseData()
             HANDLING_EVENTS = uow.newEntity( HandlingEventsEntity.class, HandlingEventsEntity.HANDLING_EVENTS_ID );
 
             logger.debug( "BASIC DATA CREATED" );
+            uow.complete();
+            }
+            catch(Exception e) {
+            	uow.discard();
+            	logger.error("CANNOT CREATE BASIC DATA");
+            }
```

Good catch!

Could you please indent the code properly and let the exception raise. If basic data cannot be created the program should fail.